LayoutTests/imported/w3c/web-platform-tests/css/css-properties-values-api/register-property-syntax-parsing-expected.txt

```diff
@@ -188,7 +188,7 @@ PASS syntax:'<length>', initialValue:'var(--moo)' is invalid
 PASS syntax:'<length>', initialValue:'10' is invalid
 PASS syntax:'<length>', initialValue:'10%' is invalid
 PASS syntax:'<length>', initialValue:'calc(5px + 10%)' is invalid
-PASS syntax:'<length>', initialValue:'calc(5px * 3px / 6px)' is invalid
+FAIL syntax:'<length>', initialValue:'calc(5px * 3px / 6px)' is invalid assert_throws_dom: function "() => CSS.registerProperty({name: name, syntax: syntax, initialValue: initialValue, inherits: false})" did not throw
```

Comment:
Do we have any history on why this was tested as invalid?

I do indeed see no reason for this to be invalid, but just want to make sure we've looked into why.

This test is also part of Interop 2024, so it's important to change if it's invalid.

Reply:
I don’t have history. I did bring it up in this issue though: https://github.com/w3c/csswg-drafts/issues/10752

Reply:
Wait, I think this is indeed invalid, dividing `px` by `px` should cancel out the units, and give a number instead of a length ? The syntax here expects a `<length>`

Reply:
It's "5px * 3px / 6px". In unit algebra that is "length * length / length" which is equal to "length * number" which is equal to "length".